securedrop_client/logic.py

```diff
@@ -753,11 +753,10 @@ def on_delete_source_success(self, result) -> None:
         # Update the sources UI.
         self.update_sources()
 
-    def on_delete_source_failure(self, result: Exception) -> None:
-        logging.info("failed to delete source at server")
```

Comment:
:+1: to remove - we do want to log when jobs fail, but we already have logging [here](https://github.com/freedomofpress/securedrop-client/blob/master/securedrop_client/queue.py#L127) (in a separate change we could add better `__repr__` that have the uuid of the item and use that).